src/main/java/org/pokecollect/dao/PokecardDAOStub.java

```diff
@@ -13,8 +13,8 @@ public class PokecardDAOStub implements IPokecardDAO{
     HashMap<Integer, Pokecard> allPokecards = new HashMap<>();
     @Override
     public Pokecard save(Pokecard pokecard) throws Exception {
-        String pokecardId = pokecard.getId();
-        allPokecards.put(Integer.parseInt(pokecardId), pokecard);
+
+        allPokecards.put(pokecard.getId(), pokecard);
```

Comment:
I changed the pokecard ID from string to int so that it will be same and it will reduce duplicated lines of code.
source: https://docs.sonarsource.com/sonarqube/9.9/user-guide/metric-definitions/#duplications